lib/resources/r4/charge_item_create.yaml

```diff
@@ -0,0 +1,425 @@
+---
+name: Parameters
+field_name_base_url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters
+fields:
+
+  - name: resourceType
+    required: 'Yes'
+    type: string
+    description: The type of the FHIR resource.
+    action: create
+    example: |
+      {
+        "resourceType": "Parameters"
+      }
+    note: resourceType must be Parameters
+    url: https://hl7.org/fhir/r4/json.html#resources
+
+  - name: parameter
+    required: 'Yes'
+    type: List of BackboneElement
+    description: Parameters defined by the operation.
+    action: create
+    url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: chargeItem
+    required: 'Yes'
+    type: resource
+    description: The ChargeItem to be created.
+    action: create
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "chargeItem",
+            "resource": {
+              "resourceType": "chargeItem"
```

Comment:
I think the resourceType should be ChargeItem with capital C

Reply:
I think we ended up going lowercase here to match the transactionId parameter. Including @exopsRamba to confirm.

Reply:
oh, I thought we wanted only the transaction name to be "chargeItem" with small C, which already is.

Reply:
So, it will remain the same.

Reply:
Sorry. I was wrong. Santanu is correct. name = chargeItem and resourceType = ChargeItem

Reply:
[changed here](https://github.com/cerner/fhir.cerner.com/pull/778/files#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR36)